### PR TITLE
Summary file now shows all solutes together in one table

### LINKED ISSUE #7980: Summary File node refactor
- [x] Instead of tables of solute data listed one after the other, arrange the solutes in a common table.
- [x] Tidy up 'Capture rules', 'Message filters', and 'Simulation Filter' sections so they fill less screen space.

### PR DESCRIPTION
**Pull request by ric394**
resolves #7980

### CODE REVIEW

**Comment by ric394** [comment]
Summary file UI now appears as below, also shows the new solute markdown table :
image

**Comment by ric394** [comment]
retest this please jenkins

**Comment by ric394** [comment]
retest this please jenkins

**Comment by ric394** [comment]
retest this please jenkins

**Comment by ric394** [comment]
retest this please jenkins

**Comment by zur003** [comment]
Any chance of reducing the number of digits displayed in the kg/ha values?

File: ApsimNG/Presenters/SummaryPresenter.cs [hunk 1]
```diff
@@ -175,14 +277,22 @@ private void UpdateView()
...
+                markdown.AppendLine(string.Join("", groupedMessages.Select(m =>
                 {
                     StringBuilder md = new StringBuilder();
+                    string soluteMsgs = "";
                     md.AppendLine($"### {m.Key.Date:yyyy-MM-dd} {m.Key.RelativePath}");
                     md.AppendLine();
                     md.AppendLine("```");
                     foreach (Message msg in m)
-                        md.AppendLine(msg.Text);
+                        if (msg.Provider.Name == "Field.Soil.NH3")
```
Comment (hol353) [inline ApsimNG/Presenters/SummaryPresenter.cs:288]: What's the purpose of looking for NH3? We don't even have a NH3 so we?

**Comment by hol353** [comment]
When I pulled your branch and ran it I got this:

image

Looking at the table can we have these columns only:

Depth   NO3 (ppm) NO3 (kg/ha)   NH4 (ppm) NH4 (kgha) 

There seems to be an extra set of columns on the right. Not sure why. There are also unneeded depth columns that I got in my screenshot.

Is it also possible to always have 3 decimal places for ppm and kgha values and have the right justified so that the decimal points all line up?

My test simulation:
[test.zip](https://github.com/APSIMInitiative/ApsimX/files/11570759/test.zip)

File: ApsimNG/Presenters/SummaryPresenter.cs (hunk 1 above)
Comment (ric394) [reply ApsimNG/Presenters/SummaryPresenter.cs:288]: I believe that is garbage code I've left behind while trying to find the correct place to modify markdown 😅. I'll remove that.

**Comment by ric394** [comment]
@hol353 Are you sure you want to remove Urea( and any other solute) from the table?
Won't there be other simulations where it will have values that users will want to view?

**Comment by hol353** [comment]
Sorry my bad - poorly written comment from me. I want all solutes in the table.

**Pull request merged**